src/App.jsx

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Clean this file so it's an empty arrow function called App, import React as well